options/locale/locale_en-US.ini

```diff
@@ -189,8 +189,8 @@ verify = Verify
 scratch_code = Scratch code
 use_scratch_code = Use a scratch code
 twofa_scratch_used = You have used your scratch code. You have been redirected to the two-factor settings page so you may remove your device enrollment or generate a new scratch code.
-twofa_passcode_incorrect = Your passcode is not correct. If you misplaced your device, use your scratch code to login.
-twofa_scratch_token_incorrect = Your scratch code is not correct.
+twofa_passcode_incorrect = Your passcode is incorrect. If you misplaced your device, you can use your scratch code to login.
```

Comment:
I would not add "you can use" since it is the only solution "If you misplaced your device"